frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal-enhanced.tsx

```diff
@@ -26,6 +26,9 @@ const DiskModalFirehoseComponent: React.FC<DiskModalFirehoseComponentProps> = (p
   const vmLikeFinal = getLoadedData(vmLikeEntityLoading, vmLikeEntity); // default old snapshot before loading a new one
   const vm = asVM(vmLikeFinal);
 
+  const allowedBusses = getOperatingSystem(vmLikeEntity).startsWith('win')
```

Comment:
I think we should rather allow all buses in the VM detail disk modal, than to hardcode some

Reply:
This is for the modal in the Disk details page.
If a user uses a windows OS it shouldn't have the option to pick `scsi` bus - just virtio or sata